app/src/main/java/com/example/dynamicmessenger/network/authorization/NetworkManager.kt

```diff
@@ -38,8 +38,8 @@ private val retrofitEro = Retrofit.Builder()
 interface JsonPlaceHolderMailExistApi {
     @Headers(MyHeaders.contentType)
     @POST(ResponseUrls.mailIsExist)
-    fun isMailExist(@Body task: EmailExistTask):
```

Comment:
`suspend` keyword can be used here. Please look at the codelab I provided to you. If this is issue we can discuss.There are more articles on this topic.